cluster/images/hyperkube/Makefile

```diff
@@ -22,7 +22,7 @@ ARCH?=amd64
 OUT_DIR?=_output
 HYPERKUBE_BIN?=$(OUT_DIR)/dockerized/bin/linux/$(ARCH)/hyperkube
 
-BASEIMAGE=k8s.gcr.io/debian-hyperkube-base-$(ARCH):0.8
```

Comment:
what happened to 0.9?

Reply:
oh, oops: https://github.com/kubernetes/kubernetes/pull/57648#issuecomment-376983024